modules/ROOT/pages/fd-reuse-config.adoc

```diff
@@ -6,11 +6,13 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can reuse a connection in a Flow Designer project instead of adding your authentication information manually. After a reusable connection is created, it is available for all Flow Designer projects in your business group. Changing your resuable configuration is can be done anytime. 
+You can reuse a connection in a Flow Designer project instead of manually adding your authentication information.
```

Comment:
How do you feel about putting the benefit right up front, something like "To save time, you can..." or "For improved efficiency, you can..."? Just a thought.

Reply:
Another idea: "...instead of manually adding your authentication information" > "...instead of repeatedly reauthenticating."

Reply:
...but if you alter the text, please BE SURE TO REREAD what you have touched. 
It is a cruel reality that in correcting one mistake, we often make some typing error that creates another....

Reply:
Going to get the typos fixes in. Will circle back on these later.